### PR TITLE
Layout edits 12-18

### PR DESCRIPTION
**Pull request by NandoJourneyman**
Moved common CSS to styles/badge-common.less 
Updates all templates in:
- dev-summit/General Public
- staging-summit/General Public

NOTE:
`#badge-artboard .badge-content` should now be `.bdg-artboard .box-content`
I was unable to change it for the 4"x6" badge, as it would break the typography size control.

### CODE REVIEW

File: src/badge_templates/default/Default/35x55/badge_1.js [hunk 1]
```diff
@@ -13,39 +13,25 @@ export default ({badge}) => {
...
             }
 
             {badge.getFeature('VIP Access') &&
-            <div id="icon-honorary-1" className="bdg-content icon-honorary">
+            <div id="icon-feature-1" className="bdg-content icon-feature">
                 <img className="bdg-image" src={feature_1_img}/>
             </div>
             }
 
-            <div className="default-label">DEFAULT BADGE</div>
```
Comment (santipalenque) [inline src/badge_templates/default/Default/35x55/badge_1.js]: Why did you remove this? This shows the user he is seeing the default badge

File: src/badge_templates/staging-summit/General Public/40x60/badge_1.js [hunk 2]
```diff
@@ -19,21 +19,21 @@ export default ({badge}) => {
             </div>
             }
 
-            <div id="name-on-badge" className="bdg-content">
+<div id="name-on-badge" className="bdg-content text-box text-box-1">
```
Comment (santipalenque) [inline src/badge_templates/staging-summit/General Public/40x60/badge_1.js]: please keep indent

File: src/badge_templates/default/Default/35x55/badge_1.js (hunk 1 above)
Comment (NandoJourneyman) [reply src/badge_templates/default/Default/35x55/badge_1.js]: Mistake. Must have copied from the 35x55.

File: src/badge_templates/staging-summit/General Public/40x60/badge_1.js (hunk 2 above)
Comment (NandoJourneyman) [reply src/badge_templates/staging-summit/General Public/40x60/badge_1.js]: Revision in progress

File: src/badge_templates/default/Default/35x55/badge_1.js (hunk 1 above)
Comment (NandoJourneyman) [reply src/badge_templates/default/Default/35x55/badge_1.js]: Fix in progress
